app/views/layouts/_footer.html.erb

```diff
@@ -3,7 +3,7 @@
     <div class="inner-footer-container">
       <a href="/">Home</a> <a href="/about">About</a> <a href="/privacy">Privacy Policy</a>
       <a href="/terms">Terms of Use</a> <a href="/contact">Contact</a> <a href="/code-of-conduct">Code of Conduct</a><br/>
-      <%= ApplicationConfig["COMMUNITY_NAME"] %> Community copyright 2016 - <%= Time.current.year %>&nbsp; 🔥
+      <%= ApplicationConfig["COMMUNITY_NAME"] %> Community copyright <%= ApplicationConfig["COMMUNITY_COPYRIGHT_START_YEAR"] %> - <%= Time.current.year %>&nbsp; 🔥
```

Comment:
the only issue I see here is that if the start year is the same of the current year, it'll look weird:

`2020 - 2020`

Maybe we can move this up to the `ApplicationHelper` and define a `copyright_year` method with that bit of logic. 

What do you think?

Reply:
Hmm I didn't think about that - this makes sense!

Reply:
Good idea @rhymes, I've used something like this before:

``ruby
def copyright_notice(start_year)
  current_year = Time.current.year 
  return "#{start_year}" if current_year == start_year
    
  "#{start_year} - #{current_year}"
end
```